tools/cbmc/proofs/Makefile.template

```diff
@@ -73,14 +63,27 @@ unpatch:
 #
 # Rules for building the CBMC harness
 
-queue_datastructure.h: $(H_OBJS_EXCEPT_HARNESS)
-	python3 @TYPE_HEADER_SCRIPT@ --binary $(FREERTOS)/freertos_kernel/queue.goto --c-file $(FREERTOS)/freertos_kernel/queue.c
+C_SOURCES = $(patsubst %.goto,%.c,$(H_OBJS_EXCEPT_HARNESS))
```

Comment:
It is both project and proof sources. I called it `C_SOURCES` because the dependencies, as written in the Makefiles, are `.goto` binaries and I'm converting them into `.c` files here.